docs/contributor-guide.md

```diff
@@ -30,22 +30,181 @@ You can see which steps will be performed by running 'dh binary --no-act' in the
+Restart      = on-failure
+NotifyAccess = main
+
+[Install]
+Alias        = storagenode-updater
+WantedBy     = multi-user.target
+```
+
+#### `Unit` section
+##### `Description=`
+Description of the service run by service unit file.
+
+##### `After=`
+Requires a certain target/service to be running before starting the current unit. `syslog.target` first creates
+the socket /dev/log and then starts the syslog daemon.
+
+#### `Service` section
+##### `Type=`
+Configures the process start-up type for this service unit. One of `simple`, `exec`, `forking`, `oneshot`, `dbus`,
+`notify` or `idle`.
+
+`Simple`: (the default if `ExecStart=` is specified but neither `Type=` nor `BusName=` are), the service manager will consider
+the unit started immediately after the main service process has been forked off. It is expected that the process
+configured with `ExecStart=` is the main process of the service. In this mode, if the process offers functionality to other
+processes on the system, its communication channels should be installed before the service is started up (e.g. sockets
+set up by `systemd`, via socket activation), as the service manager will immediately proceed starting follow-up units, right
+after creating the main service process, and before executing the service's binary.
+
+##### `User=`, `Group=`
+Set the UNIX user or group that the processes are executed as, respectively. Takes a single user or
+group name, or a numeric ID as argument.
+
+##### `ExecStart=`
+Commands with their arguments that are executed when this service is started.
+
+##### `Restart=`
+Configures whether the service shall be restarted when the service process exits, is killed, or a timeout is
+reached. The service process may be the main service process, but it may also be one of the processes specified with
+`ExecStartPre=`, `ExecStartPost=`, `ExecStop=`, `ExecStopPost=`, or `ExecReload=`.
+
+Takes one of `no`, `on-success`, `on-failure`, `on-abnormal`, `on-watchdog`, `on-abort`, or `always`.
+
+If set to `on-failure`, the service will be restarted when the process exits with a non-zero exit code, is terminated by a
+signal (including on core dump, but excluding the aforementioned four signals), when an operation (such as service reload)
+times out, and when the configured watchdog timeout is triggered.
+
+##### `NotifyAccess=`
+Controls access to the service status notification socket, as accessible via the sd_notify(3) call. Takes one of `none`
+(the default), `main`, `exec` or `all`. If `none`, no daemon status updates are accepted from the service processes, all status
+update messages are ignored.
+
+`main` - only service updates sent from the main process of the service are accepted.
+
+#### `Install` section
+##### `Alias=`
+A space-separated list of additional names this unit shall be installed under. The names listed here must have the same
```

Comment:
But the names we have listed in `Alias=` _don't_ have the same suffix as the unit filename. Is that right?

Reply:
what do you mean?

Reply:
For example, line 141 says "`Alias = storagenode-updater`" but the unit filename has the suffix `.service`. So, either line 141 should say "`Alias = storagenode-updater.service`" or this line of the guide about "`The names listed here must have the same suffix (i.e. type) as the unit filename`" is wrong.